CRM/Mailing/BAO/MailingJob.php

```diff
@@ -207,11 +213,12 @@ public static function runJobs($testParams = NULL, $mode = NULL) {
 
         $transaction = new CRM_Core_Transaction();
 
-        $saveJob = new CRM_Mailing_DAO_MailingJob();
-        $saveJob->id = $job->id;
-        $saveJob->end_date = date('YmdHis');
-        $saveJob->status = 'Complete';
-        $saveJob->save();
+        $jobParams = array(
+          'id' => $job->id,
+          'start_date' => date('YmdHis'),
```

Comment:
this is wrong - but fixed in my PR where I incorporated this change (start_date vs end_date)